pages/support.html

```diff
@@ -36,22 +36,15 @@ <h2>{% trans "Community Forums" %}</h2>
   </div>
 
   <div class="halfbox_right">
-    <h2>{% trans "Developer Mailing List" %}</h2>
```

Comment:
Can we keep the developer mailing list section but swap places with Zulip so Zulip is more prominent?